core/src/main/java/nl/nn/adapterframework/pipes/Json2XmlValidator.java

```diff
@@ -144,9 +144,11 @@ public PipeRunResult doPipe(Message input, IPipeLineSession session, boolean res
 				if (!getOutputFormat(session,responseMode).equalsIgnoreCase(FORMAT_JSON)) {
 					PipeRunResult result=super.doPipe(new Message(messageToValidate),session, responseMode);
 					if (isProduceNamespaceLessXml()) {
-						String msg=(String)result.getResult();
-						msg=XmlUtils.removeNamespaces(msg);
-						result.setResult(msg);
+						try {
+							result.setResult(XmlUtils.removeNamespaces(result.getResult().asString()));
```

Comment:
waarom geen message?

Reply:
Omdat XmlUtils.removeNamespaces geen message accepteert

Reply:
Maar dat kan je wel maken :)
Nu maak je van een message een string, en in XmlUtils.removeNamespaces weer een message.